### PR TITLE
[Horizon] - Base project

### PR DESCRIPTION
**Pull request by szabinst**
### Description

Base Horizon project. This branch will serve as a long-lived feature branch. Upcoming PRs will be merged to this branch.
 
--- 
### What to review
- General project structure,
- architecture,
- and such.

--- 

### What to ignore
- SwiftUI implementation details as they are subject to change.

### CODE REVIEW

**Comment by inst-danger** [comment]
<details><summary>Horizon Build QR Code:</summary>

</details>

**Comment by inst-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>One or more files are below the minimum test coverage 50%</td>
    </tr>
  </tbody>
</table>


Coverage | New % | Master % | Delta
-------- | ----- | -------- | -----
**Canvas iOS** | 91.35% | 91.41% | -0.06%
Core/Core/Planner/CalendarEvent/Model/API/PostCalendarEventRequest.swift | 0% | 0% | 0%
Core/Core/Planner/CalendarEvent/View/CustomFrequencyComponents/OccurrencesCountInputDialog.swift | 0% | 0% | 0%
Core/Core/Planner/CalendarEvent/Model/Helpers/RecurrenceRule+SelectionDescription.swift | 43.33% | 43.33% | 0%
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 932d6bf0a212afec9fa41638ddc6af57e1a6e5a5
</p>

**Review by vargaat — APPROVED** [review]

File: Horizon/Horizon.xcodeproj/project.pbxproj
Comment (vargaat) [inline Horizon/Horizon.xcodeproj/project.pbxproj:1]: Since we have a yml file this could go to gitignore.

Comment (szabinst) [reply Horizon/Horizon.xcodeproj/project.pbxproj:1]: Good idea, I'll make the change in a new PR.

**Pull request closed without merging**